test/acceptance/framework/flags/flags.go

```diff
@@ -93,6 +89,8 @@ func (t *TestFlags) init() {
 
 	flag.BoolVar(&t.flagUseKind, "use-kind", false,
 		"If true, the tests will assume they are running against a local kind cluster(s).")
+
+	t.flagEnterpriseLicense = os.Getenv("CONSUL_ENT_LICENSE")
```

Comment:
should we also make it possible to set via a flag?

Reply:
Done ✅